### PR TITLE
Ticket647 keithley 2700 opi files

### PR DESCRIPTION
**Pull request by esouthren**
### Description of work

OPI for Keithley 2700

### Ticket

[Ticket 647](https://github.com/ISISComputingGroup/IBEX/issues/647)

### Acceptance criteria

- Displays all required PVs. 

Notes: 

- This ticket does not include the Temperature/Drift values, they are calculated from the Resistance readings and will be completed in [Ticket 2994](https://github.com/ISISComputingGroup/IBEX/issues/2994)

---

#### Code Review

- [ ] Is the code of an acceptable quality?
- [ ] Do the changes function as described and is it robust?
- [ ] Have the changes been documented in the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev)?

### Final Steps
- [ ] Reviewer has moved the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev) entry for this ticket in the "Changes merged into master" section

### CODE REVIEW

File: base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml [hunk 1]
```diff
@@ -265,6 +265,23 @@
...
 			</value>
 		</entry>
+		<entry>
+			<key>Keithley 2700</key>
+			<value>
+				<categories>
+					<category>Multimeters</category>
+				</categories>
+				<type>KHLY2700</type>
+				<path>Keithley/Keithley2700.opi</path>
```
Comment (Tom-Willemsen) [inline base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml]: This points at an OPI which doesn't exist.

File: base/uk.ac.stfc.isis.ibex.opis/resources/Keithley2700/display_table_values.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+color = ColorFontUtil.getColorFromRGB(240,245,245)
+i = 0
+for i in range(20):
+    reading = PVUtil.getString(pvs[i])
+    #temp = PVUtil.getString(pvs[i+40])
+    #drift = PVUtil.getString(pvs[i+20])
+    
+    table.setCellText(row, 3, reading)
+    #table.setCellText(row, 2, temp)
+    #table.setCellText(row, 1, drift)
```
[the remark this answers is not in the document]
Comment (esouthren) [reply base/uk.ac.stfc.isis.ibex.opis/resources/Keithley2700/display_table_values.py]: Removed

File: base/uk.ac.stfc.isis.ibex.opis/resources/Keithley2700/display_table_values.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    
+    if row%2 != 0:
+		for col in range(0,4):
+			table.setCellBackground(row, col, color)
```
[the remark this answers is not in the document]
Comment (esouthren) [reply base/uk.ac.stfc.isis.ibex.opis/resources/Keithley2700/display_table_values.py]: Made spaces consistent

File: base/uk.ac.stfc.isis.ibex.opis/my_logs_directory/check_opi_format/TEST-check_opi_format_tests.TestCheckOpiFormatMethods-20180323115303.xml [hunk 4]
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" ?>
```
[the remark this answers is not in the document]
Comment (esouthren) [reply base/uk.ac.stfc.isis.ibex.opis/my_logs_directory/check_opi_format/TEST-check_opi_format_tests.TestCheckOpiFormatMethods-20180323115303.xml]: Removed

File: base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml (hunk 1 above)
Comment (esouthren) [reply base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml]: Modified link

**Pull request merged**